security/tor/src/opnsense/service/templates/OPNsense/Tor/torrc

```diff
@@ -62,8 +62,7 @@ Log {{ OPNsense.tor.general.logfilelevel }} file /var/log/tor.log
 Log {{ OPNsense.tor.general.sysloglevel }} syslog
 {% endif %}
 
-
-
+Schedulers KISTLite
```

Comment:
from the man page:
```
Schedulers KIST|KISTLite|Vanilla
```
So this should be a dropdown in the menu which is not requrired so if nothing is selected, the default value is used